MzML/Mzml.cs

```diff
@@ -354,6 +354,24 @@ private static MsDataScan GetMsDataOneBasedScanFromConnection(Generated.mzMLType
                 }
             }
 
+            double high = double.NaN;
```

Comment:
There's a chance that the range doesn't get parsed, which will cause a crash if the values are double.NaN. Use 0 and double.Max?

Reply:
making those suggested changes causes unit test failure